src/block.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-env mocha */
+/* globals apiClients */
+'use strict'
+
+const expect = require('chai').expect
+
+module.exports = (common) => {
+  describe.only('.block', () => {
+    const blorbKey = 'QmPv52ekjS75L4JmHpXVeuJ5uX2ecSfSZo88NSyxwA3rAQ'
+    const blorb = Buffer('blorb')
+
+    it('returns an error when putting an array of files', () => {
+      return apiClients.a.block.put([blorb, blorb], (err) => {
+        console.log(err)
+        expect(err).to.be.an.instanceof(Error)
+      })
+    })
+
+    it('block.put', (done) => {
+      apiClients.a.block.put(blorb, (err, res) => {
+        expect(err).to.not.exist
+        expect(res).to.have.a.property('Key', 'QmPv52ekjS75L4JmHpXVeuJ5uX2ecSfSZo88NSyxwA3rAQ')
+        done()
+      })
+    })
+
```

Comment:
Also, follow the same pattern of organisation that other tests have

- .block
  - callback API
    - test 1
    - test 2
    - ...
  - promises API
    - test 1
    - test 2
    - ...

See https://github.com/ipfs/interface-ipfs-core/blob/master/src/files.js for reference